package.json

```diff
@@ -21,14 +21,15 @@
   },
   "devDependencies": {
     "@netlify/plugin-nextjs": "^5.8.1",
+    "@stately-cloud/schema": "^0.30.0",
```

Comment:
Moved the Stately Schema dependency here, which eliminates the need for a separate package.json / node_modules